dev/com.ibm.ws.jpa.container.core/src/com/ibm/ws/jpa/management/JaxbPUnit.java

```diff
@@ -27,124 +27,145 @@
 
 /**
  * This interface is an abstraction of <persistence-unit> in persistence.xml. <p>
- * 
+ *
  * Get methods are provided for all attributes of <persistence-unit> for all
  * schema versions of persistence.xml (i.e. 1.0 and 2.0, etc.). This allows
  * the client of this interface to be coded (and compiled) independent of the
  * schema version. <p>
- * 
+ *
  * A different implementation of this interface will be provided for each schema
  * version of persistence.xml and will wrap the JAXB generated class that
  * represents a <persistence-unit>; generally Persistence.PersistenceUnit. <p>
- * 
+ *
  * Get methods on the generated JAXB class which return other JAXB generated
  * classes will instead return either a java primitive or javax.persistence
  * representation of that data; allowing the client of this interface to
  * be coded independent of the JAXB implementation. <p>
  **/
-abstract class JaxbPUnit
-{
+abstract class JaxbPUnit {
     private static final TraceComponent tc = Tr.register(JaxbPUnit.class,
                                                          JPA_TRACE_GROUP,
                                                          JPA_RESOURCE_BUNDLE_NAME);
 
     /**
      * Gets the value of the description property.
-     * 
+     *
      * @return value of the description property.
      */
     abstract String getDescription();
 
     /**
      * Gets the value of the provider property.
-     * 
+     *
      * @return value of the provider property.
      */
     abstract String getProvider();
 
+    /**
+     * Gets the value of the qualifier property.
+     *
+     * @return value of the qualifier property.
+     */
+    List<String> getQualifier() {
+        // Not available in 1.0, 2.0, 2.1, 2.2, 3.0 persistence schema
+        // Per the spec, must return null when not in xml
+        return null;
```

Comment:
This is causing test failures... rather than null, this should be an empty list.... like this:
            return new ArrayList<String>();

Reply:
done